ml-pipelines/README.md

```diff
@@ -0,0 +1,68 @@
+# ML Pipelines
+
+ML Pipelines is the Open Data Hub's pipeline solution for data scientists. It is built on top of the upstream [Kubeflow Piplines](https://github.com/kubeflow/pipelines) and [kfp-tekton](https://github.com/kubeflow/kfp-tekton) projects. The Open Data Hub community has a [fork](https://github.com/opendatahub-io/ml-pipelines) of this upstream under the Open Data Hub org.
+
+
+## Installation
+
+### Prerequisites
+
+1. The cluster needs to be OpenShift 4.9 or higher
+2. OpenShift Pipelines 1.7.2 or higher needs to be installed on the cluster
+3. The Open Data Hub operator needs to be installed
+4. The default installation namespace for ML Pipelines is `odh-applications`. This namespace will need to be created. In case you wish to install in a custom location, create it and update the kfdef as documented below.
+
+### Installation Steps
+
+1. Ensure that the prerequisites are met.
+2. Apply the kfdef at [kfctl_openshift_ml-pipelines.yaml](https://github.com/opendatahub-io/odh-manifests/blob/pipelines-prototype/kfdef/kfctl_openshift_ml-pipelines.yaml). You may need to update the `namespace` field under `metadata` in case you want to deploy in a namespace that isn't `odh-applications`.
```

Comment:
This should be referring the master branch so that any user is pulling a "stable" version of ml-pipelines